.changeset/cyan-geese-smell.md

```diff
@@ -0,0 +1,6 @@
+---
+"@cloudflare/workflows-shared": minor
+"miniflare": patch
```

Comment:
nit: think this should be a minor for miniflare

Reply:
yeah that's fair, i'll change it